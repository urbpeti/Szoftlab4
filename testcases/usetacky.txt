Use up a tacky
Robot R1 34 1
Robot R2 33 1
Robot R3 32 1
Robot R4 31 1
Item Tacky 35
List_Items
Step
Step
Step
Step
List_Items