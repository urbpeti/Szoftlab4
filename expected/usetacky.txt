Tacky 35.0 4
